src/NuGetForUnity/Editor/NugetHelper.cs

```diff
@@ -589,6 +602,39 @@ public static void Push(NuspecFile nuspec, string nuspecFilePath, string apiKey
             RunNugetProcess(arguments);
         }
 
+        /// <summary>
+        /// Selects the given package to be uninstalled or updated.
+        /// </summary>
+        /// <param name="package">Package to be selected.</param>
+        public static void Select(NugetPackage package)
+        {
+            selectedInstalledPackages.Add(package.Id, package);
+            SelectedPackagesCount++;
```

Comment:
It would be clearer if you would set package.Selected to true in here and set it to false in Unselect method.